demo/docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.6
+
+ENV USER=root
+
+# Upgrade or install packages needed
+RUN apt-get update && apt-get -y install \
+    g++ \
+    gcc \
+    binutils \
+    clang \ 
+    cmake 
```

Comment:
`gcc` is a dependency of `cmake`, so looks like the minimal requirements are `cmake` and `g++`, the rest we can remove